docs/manual/common/concepts/ServiceDiscovery.md

```diff
@@ -0,0 +1,30 @@
+# Service Discovery
+
+Microservice may scale up and down and also must stay mobile. In those scenarios, a Service Registry (also referred to [[Service Locator#ServiceLocator]]) is required. It contains an updated list of available microservices with a list of reachable host and ip's where each microservive can be found.
+
```

Comment:
Added a section on Service Discovery on the concepts site to explain that Lagom uses both Server-side and client-side discovery and only 3rd party registration.